### DISCUSSION TITLE
How to remove touchpad 2 finger scrolling ofsetting page?

### DISCUSSION THREAD

**Discussion by 20-buck-spin** [tauri-apps/tauri#6265] [Q&A] [answered]
I'm trying to learn the toolkit but ran into an issue with 2 finger scrolling on my laptop touchpad on the displayed page (in dev mode and in release mode). The frontend is written in vuejs using the vue router.

If I use a two finger horizontal scroll on my touch pad, that causes the usual "back animation" that we can see in chrome, and it also changes the view to the last open router page (killing any running functions and processes on the front end)he front end)

Image of the issue, the white part is not a part of the frontend
![image](https://user-images.githubusercontent.com/96288970/218666636-99f2b5d5-d68a-4c70-b340-1f66b73e6523.png)

**Comment by FabianLars** [comment, marked as answer]
Duplicate of https://github.com/tauri-apps/tauri/discussions/5958.
